{
    "url": "https://api.github.com/repos/spack/spack/issues/3588",
    "repository_url": "https://api.github.com/repos/spack/spack",
    "labels_url": "https://api.github.com/repos/spack/spack/issues/3588/labels{/name}",
    "comments_url": "https://api.github.com/repos/spack/spack/issues/3588/comments",
    "events_url": "https://api.github.com/repos/spack/spack/issues/3588/events",
    "html_url": "https://github.com/spack/spack/issues/3588",
    "id": 217627254,
    "node_id": "MDU6SXNzdWUyMTc2MjcyNTQ=",
    "number": 3588,
    "title": "Datamash build error",
    "user": {
        "login": "sknigh",
        "id": 18291531,
        "node_id": "MDQ6VXNlcjE4MjkxNTMx",
        "avatar_url": "https://avatars.githubusercontent.com/u/18291531?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/sknigh",
        "html_url": "https://github.com/sknigh",
        "followers_url": "https://api.github.com/users/sknigh/followers",
        "following_url": "https://api.github.com/users/sknigh/following{/other_user}",
        "gists_url": "https://api.github.com/users/sknigh/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/sknigh/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/sknigh/subscriptions",
        "organizations_url": "https://api.github.com/users/sknigh/orgs",
        "repos_url": "https://api.github.com/users/sknigh/repos",
        "events_url": "https://api.github.com/users/sknigh/events{/privacy}",
        "received_events_url": "https://api.github.com/users/sknigh/received_events",
        "type": "User",
        "site_admin": false
    },
    "labels": [],
    "state": "closed",
    "locked": false,
    "assignee": null,
    "assignees": [],
    "milestone": null,
    "comments": 1,
    "created_at": "2017-03-28T17:00:21Z",
    "updated_at": "2017-03-28T17:59:45Z",
    "closed_at": "2017-03-28T17:59:45Z",
    "author_association": "CONTRIBUTOR",
    "active_lock_reason": null,
    "body": "``` bash\r\n==> Building datamash [AutotoolsPackage]\r\n==> Executing phase : 'autoreconf'\r\n==> Executing phase : 'configure'\r\n==> Executing phase : 'build'\r\n==> Error: ProcessError: Command exited with status 2:\r\n    'make' '-j56'\r\nspack/lib/spack/spack/build_systems/autotools.py:271, in build:\r\n     266      def build(self, spec, prefix):\r\n     267          \"\"\"Makes the build targets specified by\r\n     268          :py:attr:``~.AutotoolsPackage.build_targets``\r\n     269          \"\"\"\r\n     270          with working_dir(self.build_directory):\r\n  >> 271              inspect.getmodule(self).make(*self.build_targets)\r\n```\r\n\r\nIt is the result of a bad symlink created at configure when building in the source directory.",
    "performed_via_github_app": null
}